terraform/environment/variables.tf

```diff
@@ -50,6 +50,7 @@ variable "accounts" {
         psql_parameter_group_family        = string
         rds_instance_type                  = string
         rds_proxy_enabled                  = bool
+        rds_proxy_routing_enabled          = bool
```

Comment:
I guess it gives more context, but are we ever likely to enable routing without the proxy being enabled? Could potentially rely on `rds_proxy_enabled` to control the various logic.